src/EditorFeatures/CSharpTest/Diagnostics/GenerateMethod/GenerateMethodTests.cs

```diff
@@ -6605,21 +6605,6 @@ private bool T()
 }");
         }
 
-        [WorkItem(774321, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/774321")]
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateMethod)]
-        public async Task TestGenerateMethodEquivalenceKey()
-        {
-            await TestEquivalenceKeyAsync(
-@"class C
-{
-    void M()
-    {
-        this.[|M1|](System.Exception.M2());
-    }
-}",
-string.Format(FeaturesResources.Generate_method_1_0, "M1", "C"));
-        }
```

Comment:
@mavasani @sharwell can either of you tell me what the original bug actually was?  I want to ensure we have a *proper* test for it.  Importantly, a test should not have any clue about thigns like 'equivalence keys'.  It's a low level internal implementation detail, and validating what that value it tells you nothing about if the fix is actually doing the right thing.

Reply:
Ah, thanks much!  Definitely a bit of a wacky test.